src/ICSharpCode.SharpZipLib/Zip/ZipConstants.cs

```diff
@@ -281,6 +281,11 @@ public static class ZipConstants
 		/// </summary>
 		public const int VersionZip64 = 45;
 
+		/// <summary>
+		/// The version required for BZip2 compression (4.6 or higher)
+		/// </summary>
+		public const int VersionBZip2 = 46;
```

Comment:
Codacy found an issue: [Change this constant to a 'static' read-only property.](https://app.codacy.com/manual/piksel/SharpZipLib/pullRequest?prid=3928649)

Reply:
Do I recall this one coming up last time codacy was turned on, where the new var style is consistent with the existing ones, but they could possibly all do with reviewing?

Reply:
Maybe, I have no idea why Codacy suddenly started doing code reviews using my account.

Reply:
ah ok